### DISCUSSION TITLE
Generate PDF Credentials for imported users

### DISCUSSION THREAD

**Discussion by ghost** [modoboa/modoboa#3213] [Help]
Hi there,

I've imported a significant amount of emails following a migration using the identity file and encrypted password. Upon importing them, they are created perfectly, but the icon to download the PDF with credentials is not generated, which does appear when I create a new one.

Do you have any ideas on how to generate or trigger the regeneration of those PDFs with credentials?

**Comment by tonioo** [comment]
PDF document are not created for imported users and there is a good reason for that: if you imported accounts with already encrypted passwords, then it is not possible to generate PDF documents since we can't display passwords in them.

**Comment by ghost** [comment]
Thanks Toonio, excellent software.

In this scenario, the users are imported as they have been migrated from one Modoboa server to another.

Is the problem related to the import process or the passwords? In other words, if I import a CSV file with passwords in plain text, would it generate the PDF for download for each identity?

**Reply by tonioo** [reply]
As of now, it won't generate PDF files even if passwords are plaintext ones
